data/settings.json

```diff
@@ -1,4 +1,6 @@
 {
     "deviceName": "@oceantroop",
+    "wifiSSID": "OrcaOne-AP",
+    "wifiPassword": "orcaoneap",
```

Comment:
Please rename `wifiSSID` to `webuiWifiSSID` and `wifiPassword` to `webuiWifiPassword`.